Sources/Nitrozen-SwiftUI/Elements/Stepper/NitrozenStepperView.swift

```diff
@@ -0,0 +1,196 @@
+        
+        Button(action: {
+            increment()
+        }) {
+            Group{
+                switch imageType {
+                case .systemImage(let name):
+                    Image(systemName: name)
+                case .assetImage(let name):
+                    Image(name)
+                }
+            }
+            .frame(
+                width: appearance.actionButtonSize.height,
+                height: appearance.actionButtonSize.height
+            )
+        }
+        .foregroundColor(titleColor)
+        .disabled(isButtonDisable)
+        .background(backGroundColor)
+        .apply(
+            shape: appearance.viewShpae,
+            color:borderColor,
+            lineWidth: appearance.actionButton.borderWidth
+        )
+        
+    }
+    
+    @ViewBuilder
+    func decrementAction() -> some View{
+        switch appearance.decrementButton {
+        case .systemImage(let imageName):
+            decrementButton(imageType: .systemImage(name: imageName))
+        case .assetImage(let imageName):
+            decrementButton(imageType: .assetImage(name: imageName))
+        }
+    }
+    
+    @ViewBuilder
+    func decrementButton(imageType:NitrozenAppearance.StepperView.CustomView) -> some View{
+        let isButtonDisable = minInputValue == Int(inputValue).or(0)
+
+        let borderColor = isButtonDisable ?
+        appearance.actionButton.borderColorDisabled : appearance.actionButton.borderColor
+        
+        let titleColor = isButtonDisable ?
+        appearance.actionButton.titleColorDisabled : appearance.actionButton.titleColor
+
+        let backGroundColor = isButtonDisable ?
+        appearance.actionButton.backgroundColorDisabled : appearance.actionButton.backgroundColor
+        
+        Button(action: {
+            decrement()
+        }) {
+            Group{
+                switch imageType {
+                case .systemImage(let name):
+                    Image(systemName: name)
+                case .assetImage(let name):
+                    Image(name)
+                }
+            }
+            .frame(
+                width: appearance.actionButtonSize.height,
+                height: appearance.actionButtonSize.height
+            )
+        }
+        .foregroundColor(titleColor)
+        .background(backGroundColor)
+        .disabled(isButtonDisable)
+        .apply(
+            shape: appearance.viewShpae,
+            color:borderColor,
+            lineWidth: appearance.actionButton.borderWidth
+        )
+    }
+    
+    func increment(){
+        if Int(inputValue).or(0) < maxInputValue{
+            var value = Int(inputValue).or(0)
+            value += step
+            if value > self.maxInputValue {
+                inputValue = min(value, self.maxInputValue).description
+            }
+            else{
+                inputValue = "\(value)"
+            }
+            
+        }
+        
+    }
+    
+    func decrement(){
```

Comment:
code modification: 

```
        if minInputValue < Int(inputValue).or(0){
            var value = Int(inputValue).or(0)
            value -= step
            inputValue = "\(value)"
        }
        else{
            inputValue = max(Int(inputValue) ?? 00, self.minInputValue).description
        }

```

Here we required else  

Case: If user enter the smaller value and try to decrease then it we required to setup max value forcefully.

Lower: 1000 
Entered Value: 100 

then it will be 1000 forcefully

Reply:
use min and max combination here in that case to keep value in range